### PR TITLE
Add new cask `mindmac`

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by singularitti** [comment]
`brew audit` was not successful but this cask has an official webpage. So not a GitHub repo.

File: Casks/m/mindmac.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+cask "mindmac" do
+  version "1.8.0"
+  sha256 "ef9b8078d59f0f8786682301b284f678699be7299784ccb56405a44a5f25d688"
+
+  url "https://github.com/MindMacApp/MindMac/releases/download/#{version}/MindMac_#{version}.dmg",
+      verified: "https://github.com/MindMacApp/MindMac/"
+  name "MindMac"
+  desc "ChatGPT client"
+  homepage "https://mindmac.app/index.html"
```
Comment (bevanjkay) [inline Casks/m/mindmac.rb:9]:
```suggestion
  homepage "https://mindmac.app"
```

**Review by bevanjkay — DISMISSED** [review]

File: Casks/m/mindmac.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  livecheck do
+    url "https://github.com/MindMacApp/MindMac"
+    strategy :github_latest
+  end
+
```
Comment (miccal) [inline Casks/m/mindmac.rb:15]:
```suggestion
depends_on macos: ">= :Ventura"

```

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @singularitti.

**Comment by singularitti** [comment]
Thank you guys!

**Pull request merged**